blueprints.yaml

```diff
@@ -321,4 +321,23 @@ form:
       label: Visitors history
       default: 20
 
+    Editor:
+      type: section
+      title: Editor
```

Comment:
To me makes more sense being `Frontmatter` as this is what it refers to at the moment. If we expand in the future to include other settings then I agree we could update the terms but as it is now I find it less confusing if it was referencing to just the frontmatter.

Eventually we could give the section a name of "Pages" and something like "Default Frontmatter Page Headers" for the label.